### PR TITLE
Update msi builder

### PR DESCRIPTION
**Pull request by twangboy**
- Update top banner
- Add icon.ico to project
- Fix icon not being used in Add/Remove Programs

### CODE REVIEW

File: build.cmd.ps1 [hunk 1]
```diff
@@ -16,14 +16,14 @@ if (-not ($?)) {exit(1)}
...
-  exit(1)
-}
+#if (-not ($displayversion -match '^[\d\.]{6,18}$')) {
+#  Write-Host -ForegroundColor Red $displayversion is not a version
+#  exit(1)
+#}
+#if (-not ($internalversion -match '^\d\d\.[\d]{1,2}\.[\d]{1,2}$')) {
+#  Write-Host -ForegroundColor Red $internalversion is not a valid msi version
+#  exit(1)
+#}
```
Comment (twangboy) [inline build.cmd.ps1]: Need to figure out what we want to do here

File: build.cmd.ps1 [hunk 2]
```diff
@@ -38,15 +38,16 @@ if ($salt_targetplatform -eq 0) {
...
+$saltpythonversion = 3
+## Detecting Python 2 or 3 from NSIS exe ...
+#$saltpythonversion=0
+#if (Test-Path ..\salt\pkg\windows\installer\Salt-Minion*Py2*.exe) {$saltpythonversion=2}
+#if (Test-Path ..\salt\pkg\windows\installer\Salt-Minion*Py3*.exe) {$saltpythonversion=3}
+#if ($saltpythonversion -eq 0) {
+#  $saltpythonversion = 2
+#  Write-Host -ForegroundColor Red "Cannot determine Python 2 or 3"
+#  exit(1)
+#}
```
Comment (twangboy) [inline build.cmd.ps1]: Is this the right fix here?

Comment (markuskramerIgitt) [reply build.cmd.ps1]: No: the Py2 build support should continue for a long time and for a number of reasons. 
Only a very limited group of experts use it, and I still depend on it.

Could you please revert?

File: build.cmd.ps1 (hunk 1 above)
Comment (markuskramerIgitt) [reply build.cmd.ps1]: Both regexp checks are a question of taste.
They serve to "fail fast" on bad input.

The msi compiler will stop and complain an illegally formed `$internalversion` with a clear and precise error message, so remove the check here, if you want to save lines of code.

Comment (markuskramerIgitt) [reply build.cmd.ps1]: Could you please paste the error messages when you use the 3000 versioning?

File: build.cmd.ps1 (hunk 2 above)
Comment (markuskramerIgitt) [reply build.cmd.ps1]: Hi @twangboy 
With https://github.com/markuskramerIgitt/salt-windows-msi/commit/0aec90c5884adcdea6bd5fac90fd7569a3684c65 I tested on 

    Salt-Minion-3000-Py3-AMD64-Setup.exe
    Salt-Minion-3000-Py2-AMD64-Setup.exe
    Salt-Minion-3000-Py3-x86-Setup.exe

Please revert your commit.

File: build.cmd.ps1 (hunk 1 above)
Comment (markuskramerIgitt) [reply build.cmd.ps1]: Same here
With https://github.com/markuskramerIgitt/salt-windows-msi/commit/0aec90c5884adcdea6bd5fac90fd7569a3684c65 I tested the msi build on 

    Salt-Minion-3000-Py3-AMD64-Setup.exe
    Salt-Minion-3000-Py2-AMD64-Setup.exe
    Salt-Minion-3000-Py3-x86-Setup.exe

Please revert your commit, not need for a change.

File: wix.d/MinionMSI/Product.wxs [hunk 3]
```diff
@@ -177,7 +177,7 @@
     <Property Id="ARPURLINFOABOUT"          Value="https://saltstack.com/community/"/>
     <Property Id="ARPURLUPDATEINFO"         Value="https://docs.saltstack.com/en/latest/topics/releases/"/>
     <Property Id="ROOTDRIVE"                Value="C:\"/>
-    <Property Id="ARPPRODUCTICON"           Value="icon"/>
+    <Property Id="ARPPRODUCTICON"           Value="icon.ico" />
```
Comment (markuskramerIgitt) [inline wix.d/MinionMSI/Product.wxs]: Nice, this is a clever simplification.

File: wix.d/MinionMSI/Product.wxs [hunk 4]
```diff
@@ -322,7 +322,7 @@
       </Feature>
     </Feature>
 
-    <Icon Id="icon" SourceFile="$(var.dist)\..\installer\salt.ico" />
+    <Icon Id="icon.ico" SourceFile="salt.ico" />
```
Comment (markuskramerIgitt) [inline wix.d/MinionMSI/Product.wxs]: Nice, this is a clever simplification.

**Pull request merged**